src/control/script_command_interface.cpp

```diff
@@ -226,6 +226,32 @@ bool ScriptCommandInterface::endToolContact()
   return server_.write(client_fd_, buffer, sizeof(buffer), written);
 }
 
+bool ScriptCommandInterface::setFrictionCompensation(const bool friction_compensation_enabled)
+{
+  // ToDo: Increase to 5.23.0 once released
+  robotVersionSupportsCommandOrWarn(urcl::VersionInformation::fromString("5.22.0"),
```

Comment:
5.23 has been released 
```suggestion
  robotVersionSupportsCommandOrWarn(urcl::VersionInformation::fromString("5.23.0"),
```